src/compiler/crystal/compiler.cr

```diff
@@ -18,6 +18,7 @@ module Crystal
   # optionally generates an executable.
   class Compiler
     CC = ENV["CC"]? || "cc"
+    CL = "cl"
```

Comment:
Why not use `ld` instead of `cc`? I don't know, but I think it'd beyond the scope of this PR. If we change to use the linker directly instead of the compiler directly we should do that on all platforms simultaneously.